core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -807,6 +807,19 @@ public ByteSequence getColumnVisibilityData() {
     return new ArrayByteSequence(colVisibility);
   }
 
+  /**
+   * Writes the column visibility into the given <code>ArrayByteSequence</code>. This method gives
+   * users control over allocation of ArrayByteSequence objects by copying into the passed in
+   * ArrayByteSequence.
+   *
+   * @param cv <code>ArrayByteSequence</code> object to copy into
+   * @return the <code>ArrayByteSequence</code> that was passed in
```

Comment:
This also needs a since tag

```suggestion
   * @return the <code>ArrayByteSequence</code> that was passed in
   * @since 3.1.0
```

Reply:
Addressed in a0cc0d7